packages/core/src/factory.ts

```diff
@@ -62,18 +62,18 @@ export function factory<Resource, Input, Output, Config extends IPrismConfig>(
     config: Config,
     validations: IPrismDiagnostic[]
   ): TE.TaskEither<Error, ResourceAndValidation & { output: Output }> => {
+    const mockCall = components.mock({
+      resource,
+      input: { data, validations },
+      config: config.mock,
+    });
+
     const produceOutput = isProxyConfig(config)
       ? components.forward(
```

Comment:
If the intention was to increase readability then I'd vote on doing also `const forwardCall = ...`. Or leaving both. as they were.

Reply:
2297a1b